.buildkite/pipeline.yml

```
@@ -1,4 +1,8 @@
 steps:
+  - label: ":shell: Tests"
+    plugins:
+      - plugin-tester#v4.1.1: ~
```

https://github.com/buildkite-plugins/plugin-tester-buildkite-plugin/tags
```suggestion
      - plugin-tester#v1.1.1: ~
```